src/Microsoft.DotNet.VersionTools/project.json

```diff
@@ -0,0 +1,16 @@
+{
+  "dependencies": {
+    "Microsoft.Build.Framework": "0.1.0-preview-00022",
+    "Microsoft.Build.Tasks.Core": "0.1.0-preview-00022",
+    "Microsoft.Build.Utilities.Core": "0.1.0-preview-00022",
+    "Microsoft.NETCore.Runtime.CoreCLR": "1.0.2",
```

Comment:
Why?  This is just a library, there shouldn't need to be a dependency on CoreCLR.